python/singa/sonnx.py

```diff
@@ -319,14 +320,21 @@ def output_name(op):
                                              pads=p,
                                              strides=s))
         elif (isinstance(op, autograd._BatchNorm2d)):
-            print(op,op.src)
+            #[(<singa.autograd.Sigmoid object at 0x7fd5ec09cb90>, 140556764852432, None, False),
+            # (<singa.autograd.Dummy object at 0x7fd5ec09c390>, 140556764824208,
+            # <singa.tensor.Tensor object at 0x7fd5ec09c290>, True),
```

Comment:
what are these??

Reply:
singa operation information.
include output variables and input variables.
I want to show that some of the dummy variables do not have tensor values